pkg/3scale/amp/component/system.go

```diff
@@ -1341,3 +1446,117 @@ func (system *System) appDeveloperPorts() []v1.ContainerPort {
 
 	return ports
 }
+
+func (system *System) systemInit(containerImage string) []v1.Container {
+	if system.Options.SystemDbTLSEnabled {
+		return []v1.Container{
+			{
+				Name:  "set-permissions",
+				Image: containerImage, // Minimal image for chmod
+				Command: []string{
+					"sh",
+					"-c",
+					"cp /tls/* /writable-tls/ && chmod 0600 /writable-tls/*",
+				},
+				VolumeMounts: []v1.VolumeMount{
+					{
+						Name:      "tls-secret",
+						MountPath: "/tls",
+						ReadOnly:  true,
+					},
+					{
+						Name:      "writable-tls",
+						MountPath: "/writable-tls",
+						ReadOnly:  false, // Writable emptyDir volume
+					},
+				},
+			},
+		}
+	} else {
+		return []v1.Container{}
+	}
+}
+
+func (system *System) sidekiqInit(containerImage string) []v1.Container {
+	if system.Options.SystemDbTLSEnabled {
+		return []v1.Container{
+			{
+				Name:  "set-permissions",
+				Image: containerImage, // Minimal image for chmod
+				Command: []string{
+					"sh",
+					"-c",
+					"cp /tls/* /writable-tls/ && chmod 0600 /writable-tls/*",
+				},
+				VolumeMounts: []v1.VolumeMount{
+					{
+						Name:      "tls-secret",
+						MountPath: "/tls",
+						ReadOnly:  true,
+					},
+					{
+						Name:      "writable-tls",
+						MountPath: "/writable-tls",
+						ReadOnly:  false, // Writable emptyDir volume
+					},
+				},
+			},
+			{
+				Name:  SystemSideKiqInitContainerName,
+				Image: containerImage,
+				Command: []string{
+					"bash",
+					"-c",
+					"bundle exec sh -c \"until rake boot:redis && curl --insecure --output /dev/null --silent --fail --head http://system-master:3000/status; do sleep $SLEEP_SECONDS; done\"",
```

Comment:
why are we adding the --insecure here? It seems to be working fine without it...but maybe I'm doing something wrong,.

Reply:
Yes it works now. Issue hit by me and @valerymo , guessing it was an issue with the latest image.